include/proxy-wasm/exports.h

```diff
@@ -149,6 +149,10 @@ Word wasi_unstable_random_get(Word, Word);
 Word pthread_equal(Word left, Word right);
 void emscripten_notify_memory_growth(Word);
 
+Word set_effective_context(Word context_id);
+void setLimitedEffectiveContext(ContextBase *context);
```

Comment:
This "limited effective context" is an entirely new concept added without any explaination.

Also, the function name doesn't follow the existing naming convention for exports, and it's not even an export function, since it's operates on host-side objects, so it shouldn't be in the `exports` namespace.